orchestration/dap_orchestration/repositories/dev_repositories.py

```diff
@@ -1,8 +1,33 @@
 from dagster import PipelineDefinition, repository
+from dagster_gcp.gcs import gcs_pickle_io_manager
+from dagster_utils.resources.beam.k8s_beam_runner import k8s_dataflow_beam_runner
+from dagster_utils.resources.beam.local_beam_runner import local_beam_runner
+from dagster_utils.resources.beam.dataflow_beam_runner import dataflow_beam_runner
+from dagster_utils.resources.google_storage import google_storage_client
 
-from dap_orchestration.repositories.base_repositories import all_jobs
+from dap_orchestration.config import preconfigure_resource_for_mode
+from dap_orchestration.pipelines import refresh_data_all, refresh_data_sample
+from dap_orchestration.resources import refresh_directory, outfiles_writer, api_token
 
 
 @repository
 def repositories() -> list[PipelineDefinition]:
-    return all_jobs()
+    return [refresh_data_all.to_job(resource_defs={
+        "extract_beam_runner": preconfigure_resource_for_mode(k8s_dataflow_beam_runner, "dev_extract"),
+        "transform_beam_runner": preconfigure_resource_for_mode(k8s_dataflow_beam_runner, "dev_transform"),
+        "refresh_directory": refresh_directory,
+        "outfiles_writer": outfiles_writer,
+        "api_token": preconfigure_resource_for_mode(api_token, "dev"),
+        "io_manager": preconfigure_resource_for_mode(gcs_pickle_io_manager, "dev"),
+        "gcs": google_storage_client,
+    }),
+        refresh_data_sample.to_job(resource_defs={
```

Comment:
Do we still want this job?

Reply:
no, removing this as we were using it to debug.
My last run of that job did successfully run the sample pipeline and copied a file to the destination bucket.